### PR TITLE
doc: update the readme of SDK and bump version to 0.0.2

### PR DESCRIPTION
**Pull request by dongx1x**
- Add quote usage example for SDK, update some descriptions
- Bump version to 0.0.2

### CODE REVIEW

File: sdk/python3/README.md [hunk 1]
```diff
@@ -37,24 +42,79 @@ There are three major functionalities provided in this SDK:
 
 ### Quote
 
+Using this SDK, user could fetch the quote from different platforms, the service detect the platform automatically and return the type and the quote.
+
+##### Quote type for platform
```
Comment (Ruoyu-y) [inline sdk/python3/README.md:47]: "#####" -> "####"

File: sdk/python3/README.md [hunk 2]
```diff
@@ -90,17 +150,17 @@ container_measurement = Measurement.get_container_measurement()
 
 ### Event log
 
-Using this sdk, user can fetch the event logs to assist the attestation/verification process. It also enables two different categories of event logs - for the platform or for a single workload/container.
+Using this SDK, user can fetch the event logs to assist the attestation/verification process. It also enables two different categories of event logs - for the platform or for a single workload/container.
 From platform perspective, it can support different Trusted Execution Environment and TPM. This sdk can also do fetching on certain number of event logs.
 
 ##### EventlogType for platform
```
Comment (Ruoyu-y) [inline sdk/python3/README.md:156]: can you also fix this one for me? "#####"->"####"

File: sdk/python3/README.md [hunk 3]
```diff
@@ -1,10 +1,15 @@
-# Confidential Cloud-Native Primitives client library for Python
+# Confidential Cloud-Native Primitives SDK for Python
 
-The Confidential Cloud-Native Primitives (CCNP) project is the solution targeted on simplify the use of Trusted Execution Environment (TEE) in cloud native environment. The service supports attestation, measurement fetching and event log collecting of various platforms including Intel Trusted Domain Extensions (TDX), Trusted Platform Modules (TPM) and AMD SEV-SNP. More platforms will be supported later.
+The Confidential Cloud-Native Primitives (CCNP) project is the solution targeted on simplifying the use of Trusted Execution Environment (TEE) in cloud-native environment. Currently, there are 2 parts included in CCNP, the services and the SDK.
+
+- Service (DaemonSet) is designed to hide the complexity of different TEE platforms and provides common interfaces and scalability for cloud-native environment.
```
Comment (Ruoyu-y) [inline sdk/python3/README.md:5]: Service (DaemonSet) is -> Service is
Since there might be other deployment mode later.

**Review by Ruoyu-y — CHANGES_REQUESTED** [review]

**Review by Ruoyu-y — COMMENTED** [review]
And also please help add 'TBA' for end-to-end examples section

File: sdk/python3/README.md (hunk 1 above)
Comment (dongx1x) [reply sdk/python3/README.md]: Done.

File: sdk/python3/README.md (hunk 2 above)
Comment (dongx1x) [reply sdk/python3/README.md]: Done.

**Review by Ruoyu-y — APPROVED** [review]
LGTM

**Pull request merged**